pkg/stream/input_sqs.go

```diff
@@ -47,10 +48,23 @@ func NewSqsInput(config cfg.Config, logger mon.Logger, s SqsInputSettings) *sqsI
 		Backoff:           s.Backoff,
 	})
 
-	return NewSqsInputWithInterfaces(logger, queue, s)
+	var unmarshaller UnmarshallerFunc
+
+	switch s.Unmarshaller {
+	case UnmarshallerMsg:
+		unmarshaller = MessageUnmarshaller
+	case UnmarshallerRaw:
+		unmarshaller = RawUnmarshaller
+	case UnmarshallerSns:
+		unmarshaller = SnsUnmarshaller
+	default:
+		logger.Fatal(fmt.Errorf("unknown unmarshaller %s", s.Unmarshaller), "")
+	}
+
```

Comment:
would be better if we store the different unmarshallers in a map inside the stream package..  
makes it open for extension, too..